tests/snap_test.go

```diff
@@ -11,13 +11,18 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-var start = time.Now()
-
 const (
 	otbrSnap     = "openthread-border-router"
 	otbrSetupApp = "openthread-border-router.otbr-setup"
 	otbrAgentApp = "openthread-border-router.otbr-agent"
 	otbrWebApp   = "openthread-border-router.otbr-web"
+
+	infraInterface = "INFRA_IF"
+)
+
+var (
+	start          = time.Now()
+	InfraInterface = "wlan0"
```

Comment:
A few issues with `InfraInterface` variable:
- It is defined as global and then used and changed inside one test function. Changing global variables is usually frowned upon for good reasons. If the intention is to set the default on top, then it should be a constant with an appropriate name, such as `defaultInfraInterface`.
- It is exported.
- The name is confusing because it is similar to `infraInterface`.

Reply:
Updated these variables to `infraInterfaceEnv`, `defaultInfraInterface` and `infraInterface`.